packages/suite/src/components/suite/AppNavigationPanel/components/AppNavigationPanelInner.tsx

```diff
@@ -74,7 +74,7 @@ const AppNavigationPanelInner = (props: Props) => (
                     <TitleRow>
                         <Title noMargin>{props.title}</Title>
                         <Aside>
-                            {props.ticker && props.ticker}
+                            {props.ticker}
```

Comment:
cool, just please notice that it's on more places (e.g. one line below)

Reply:
no need to change right now, we could find those using some regex in the whole project and change it in some separate commit